repository/src/raw/mod.rs

```diff
@@ -163,7 +163,8 @@ pub trait RawRepository: Send + Sync + 'static {
         max: Option<usize>,
     ) -> Result<Vec<CommitHash>, Error>;
 
-    /// Queries the commits from `ancestor` to `descendant`, including both.
+    /// Queries the commits from the very next commit of `ancestor` to `descendant`.
+    /// `ancestor` not included, `descendant` included.
```

Comment:
Please add this :
`It fails if the two commits are the same`.

Reply:
fixed